integration-cli/docker_cli_events_test.go

```diff
@@ -134,10 +128,9 @@ func (s *DockerSuite) TestEventsLimit(c *check.C) {
 
 func (s *DockerSuite) TestEventsContainerEvents(c *check.C) {
 	dockerCmd(c, "run", "--rm", "busybox", "true")
-	eventsCmd := exec.Command(dockerBinary, "events", "--since=0", fmt.Sprintf("--until=%d", daemonTime(c).Unix()))
-	out, exitCode, err := runCommandWithOutput(eventsCmd)
-	if exitCode != 0 || err != nil {
-		c.Fatalf("Failed to get events with exit code %d: %s", exitCode, err)
+	out, exitCode := dockerCmd(c, "events", "--since=0", fmt.Sprintf("--until=%d", daemonTime(c).Unix()))
+	if exitCode != 0 {
```

Comment:
nit, this execCode is always != 0 here since there's no error here (otherwise dockerCmd should have failed before)